examples/README.md

```diff
@@ -7,3 +7,23 @@ In this folder there are examples taken from the delta.io quickstart guide and d
 ### Instructions
 * To run an example in Python run `spark-submit --packages io.delta:delta-core_2.12:{Delta Lake version} PATH/TO/EXAMPLE`
 * To run the Scala examples, `cd examples/scala` and run `./build/sbt "runMain example.{Example class name}"` e.g. `./build/sbt "runMain example.Quickstart"`
+
```

Comment:
Please update the instruction at the top to make sure it is clear that there are two ways of running the examples.

Reply:
Can you run the scala examples inside docker as well?